src/content/docs/logs/logs-context/configure-logs-context-apm-agents.mdx

```diff
@@ -23,7 +23,7 @@ redirects:
   - /docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-apm-agents
 ---
 
-When you need to correlate log data with other telemetry data, enable logs in context in New Relic. Logs in context functionality adds metadata that links your logs with related APM data, like [errors](/docs/apm/apm-ui-pages/error-analytics/errors-page-find-fix-verify-problems/) or [distributed traces](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data), or your platform performance data from [infrastructure monitoring logs](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) in New Relic One.
+Enabling New Relic's **logs in context** links and correlates your log data with other New Relic-monitored app and infrastructure data. This means that, when applicable, you'll see logs data or links to log data in other New Relic UI experiences, such as our APM UI, infrastructure monitoring UI, distributed tracing, errors inbox, and more.  
```

Comment:
I like this change!

Reply:
i just submitted another edit actually if you want to re-take a look.